collections/retail/general-notifications/event-general-notifications/phone-number-deleted/event-general-notifications.json

```diff
@@ -0,0 +1,28 @@
+{
+  "eventClassName": "com.backbase.dbs.user.manager.event.spec.v2.DeletePhoneNumberSecNotEvent",
+  "id": "phone-number-deleted",
+  "eventBusinessName": "Phone Number Deleted",
+  "recipientType": "internal",
+  "extractor": "#this['userInternalId']",
+  "generalNotifications": [
+    {
+      "id": "phone-number-deleted",
+      "name": "Phone number deleted",
```

Comment:
@AnjBB72 seems this GN does not have preferences, right? Please add userPreferenceEnabled field here. E.g:
"userPreferenceEnabled": false,

Example: https://github.com/Backbase/engagements-data/blob/master/collections/retail/general-notifications/event-general-notifications/message-received/event-general-notifications.json